packages/pancake-uikit/src/components/PancakeToggle/StyledPancakeToggle.tsx

```diff
@@ -0,0 +1,171 @@
+import styled from "styled-components";
+import { scales, PancakeToggleProps, HandleProps, InputProps, ScaleKeys } from "./types";
+
+const scaleKeyValues = {
+  sm: {
+    handleSize: "16px",
+    travelDistance: "16px",
+    toggleHeight: "20px",
+    toggleWidth: "36px",
+    pancakeThickness: "1px",
+    pancakeTwoOffset: "0px",
+    pancakeThreeOffset: "-3px",
+    butterTop: "3px",
+    butterLeft: "10px",
+    butterWidth: "6px",
+    butterHeight: "5px",
+    butterThickness: "0.5px",
+    butterRadius: "2px",
+    butterSmearOneTop: "10px",
+    butterSmearOneLeft: "2.5px",
+    butterSmearTwoTop: "11px",
+    butterSmearTwoRight: "2.5px",
+  },
+  md: {
+    handleSize: "32px",
+    travelDistance: "34px",
+    toggleHeight: "40px",
+    toggleWidth: "72px",
+    pancakeThickness: "2px",
+    pancakeTwoOffset: "-3px",
+    pancakeThreeOffset: "-8px",
+    butterTop: "3px",
+    butterLeft: "16px",
+    butterWidth: "12px",
+    butterHeight: "11px",
+    butterThickness: "1px",
+    butterRadius: "4px",
+    butterSmearOneTop: "20px",
+    butterSmearOneLeft: "5px",
+    butterSmearTwoTop: "22px",
+    butterSmearTwoRight: "5px",
+  },
+};
+
+const getScale = (property: ScaleKeys) => ({ scale = scales.MD }: PancakeToggleProps) => {
+  return scaleKeyValues[scale][property];
+};
+
+export const PancakeStack = styled.div<HandleProps>`
+  position: relative;
+  display: inline-block;
+
+  &:label:before {
+    content: none;
+  }
+
+  .pancakes {
+    transition: 0.6s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+  }
+
+  .pancake {
+    background: #e27c31;
+    border-radius: 50%;
+    width: ${getScale("handleSize")};
```

Comment:
I could be wrong here... but I _think_ `styled-system` components support an array of breakpoint-based props?  https://styled-system.com/guides/array-props. That might be a solution here? If these are all divs, then restyling the `Box` component ought to give you access to them?

Reply:
Yeah but we'd need the size for `label` and `input` too. But actually let me try if pretending as if everything's a div would work or not. 🤔

Reply:
Ah of course, not everything _is_ a div. Other option would be to make some styled-system uikit versions of those html elements? `Box` & `Flex` are straightforward styled divs https://github.com/pancakeswap/pancake-toolkit/blob/6c87947f271aa8742dcb8ea5c119bc0f1a50d685/packages/pancake-uikit/src/components/Box/Box.tsx#L5

Reply:
eh, just tried to do similar thing and while it might work for some simple stuff - here there is just too many different values between small and big version. Even if I'd plug in styled-system props to do `width = ['36px', '72px']`, I still would need to derive somehow all other values. Would be much better to make custom `scale = ['sm', 'md']` prop that does it all behind the scenes. 🤔 

Anyway, let's maybe do it as a separate PR? Cause if we do something like that here it would make sense to refactor the `Toggle` component too.
```
sm: {
    handleSize: "16px",
    travelDistance: "16px",
    toggleHeight: "20px",
    toggleWidth: "36px",
    pancakeThickness: "1px",
    pancakeTwoOffset: "0px",
    pancakeThreeOffset: "-3px",
    butterTop: "3px",
    butterLeft: "10px",
    butterWidth: "6px",
    butterHeight: "5px",
    butterThickness: "0.5px",
    butterRadius: "2px",
    butterSmearOneTop: "10px",
    butterSmearOneLeft: "2.5px",
    butterSmearTwoTop: "11px",
    butterSmearTwoRight: "2.5px",
  },
  md: {
    handleSize: "32px",
    travelDistance: "34px",
    toggleHeight: "40px",
    toggleWidth: "72px",
    pancakeThickness: "2px",
    pancakeTwoOffset: "-3px",
    pancakeThreeOffset: "-8px",
    butterTop: "3px",
    butterLeft: "16px",
    butterWidth: "12px",
    butterHeight: "11px",
    butterThickness: "1px",
    butterRadius: "4px",
    butterSmearOneTop: "20px",
    butterSmearOneLeft: "5px",
    butterSmearTwoTop: "22px",
    butterSmearTwoRight: "5px",
  },
```

Reply:
Holy shit sorry I thought sm/md here were referring to breakpoints (and dynamically resizing the component based on those), rather than a prop that you pass down to determine a range of component sizes.

Reply:
Anyway, lgtm! I'd maybe suggest breaking the massive style string literal into separate styled components where possible but 🤷.  I'll leave the others to feedback 👍

Reply:
I am inclined to agree with @ChefHutch but in this case because it is a specific one-off component this works as well.